sim_pipeline/galaxy_galaxy_lens_pop.py

```diff
@@ -110,11 +105,11 @@ def __init__(
         self.cosmo = cosmo
         self.f_sky = sky_area
 
-    def select_lens_at_random(self, **kwargs_lens_cut):
+    def generate_random_lensed_system(self, **kwargs_lens_cut):
```

Comment:
was there a specific reason to change this? The reason I am asking is as multiple other people are using the sim_pipeline this is again not backwards compatible and might create some hassle

Reply:
It was to make it clear to the user that they should be generating a `lensed_system` object. I thought that `select_lens_at_random` wasn't clear as to whether they should implement the method such that it returns a deflector or a galaxy_galaxy_lens type object (i.e a lensed system that has mapped a deflector to a source).

Reply:
Hi @padma18-vb, eventually we are going to use lens and LensPop as a general term for all lensed system and lens system population respectively. So, I think lens will make more sense then!

Reply:
Ah, okay! In that case, I think I will rename the base classes as well:

- `LensedSystem` --> `Lens`
- `LensedPopulation` --> `LensPop`

Does that sound okay?

Reply:
Maybe we should wait re-naming them, certainly Lens and Deflector could be ambiguous for some users, which we should avoid

Reply:
Yes, we should wait for them.